hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -89,6 +91,7 @@ public class SnapshotDeletingService extends AbstractKeyDeletingService {
   // from the same table and can send deletion requests for same snapshot
   // multiple times.
   private static final int SNAPSHOT_DELETING_CORE_POOL_SIZE = 1;
+  private static final int MIN_ERR_LIMIT_PER_TASK = 1000;
```

Comment:
Are we sure that 1000 entries would be well under the Ratis Byte limit or should we calculate this based on the remainingBufLimit?

Reply:
32MB is limit set default, but even 6000* (4KB as approx size taken) => 24MB crosses, then we are going with 1000, i.e. 32MB/1000 => 32KB.
We can not do calculation for size till its serialize, and will have impact like binary calculation till we achieve the meeting criteria. This kind of thing is not required, and even if 1000 entry, limit is not met, user need increase buffer size of Ratis.

So this logic is not done, as its not simple to identify remaining buf directly without serializing.

Reply:
Thanks for explaining.